deploy/cluster_role_binding.yaml

```diff
@@ -7,6 +7,7 @@ roleRef:
   kind: ClusterRole
   name: business-automation-operator
 subjects:
-- kind: ServiceAccount
+- apiGroup: rbac.authorization.k8s.io
```

Comment:
why this addition here?

Reply:
Was a run with a different operator-sdk version, fixed